birdhouse/components/monitoring/config/magpie/config.yml.template

```diff
@@ -0,0 +1,48 @@
+providers:
+  grafana:
+    url: http://grafana:3000
+    title: Grafana
+    public: true
+    c4i: false
+    type: api
+    sync_type: api
+  prometheus:
+    url: http://prometheus:9090
+    title: Prometheus
+    public: true
+    c4i: false
+    type: api
+    sync_type: api
+  alertmanager:
+    url: http://alertmanager:9093
+    title: AlertManager
+    public: true
+    c4i: false
+    type: api
+    sync_type: api
```

Comment:
Just curious if this works (UI-wise, everything responds/reacts correctly)?
Have you tried validating access to those endpoints?

Reply:
As of yesterday, no ...I was still working on it but everything should be working now. I'm not too familiar with what these pages should necessarily look like so if @tlvu can take a look and let me know if anything looks off, I'd appreciate it.